library/timing.c

```diff
@@ -266,18 +267,16 @@ unsigned long mbedtls_timing_get_timer( struct mbedtls_timing_hr_time *val, int
 /* It's OK to use a global because alarm() is supposed to be global anyway */
 static DWORD alarmMs;
 
-static DWORD WINAPI TimerProc( LPVOID TimerContext )
+static void TimerProc( void *TimerContext )
 {
-    ((void) TimerContext);
+    (void)TimerContext;
```

Comment:
Please adhere to the coding style present before, that is ((void) TimerContext);

Reply:
> that is ((void) TimerContext);

Space is necessary, but probably outer parentheses were not? For example, lines 347, 386, 392

Reply:
Two of these examples are function calls, and a quick grep among the sources shows that double parentheses are a majority. I won't be pushing for this change however, since it's minor - we'll leave it to other reviewers' opinion.